beps/0001-notifications-system/README.md

```diff
@@ -85,44 +95,58 @@ The role of the notifications plugin is to manage the lifecycle of notifications
 The notification backend stores notification using the [database service](https://backstage.io/docs/backend-system/core-services/index#database). In particular it needs to store the following information for each notification:
 
 - ID
-- Title
-- Priority
-- Origin
-- Topic (optional)
-- Creation timestamp (generated)
-- Receivers
-- Read status by user
-- Payload
-  - icon (optional)
-  - message (optional)
-  - links (list, optional)
-
-The receivers is **not** a list of users, but rather a filter that describes who should receive the notification. It must be possible to evaluate this filter in a database query, so that we can efficiently fetch all notifications for a given user. The same filter will also be used by the signal backend to determine which users should receive a signal.
-
-The title is mandatory human readable text shortly describing the notifications.
-
-The priority is one of the predefined values to help with presentation of the notifications or their filtering. Values: Critical, High, Normal, Low.
+- Read date
+- Done date
```

Comment:
What is `Done` for? Do we build a workflow on top of notifications?

Reply:
Done is separate from `read`. When the user navigates to the link provided by the notification, it is marked read. Users can also mark the notification done manually which also makes it read.

Reply:
It assumes that a natification is task. I don't think this is correct.

Reply:
No it doesn't. Think about GitHub notifications, they also have `read` and `done` as separated

Reply:
Hmm, but that's kind of exactly what GitHub does. It treats the notifications as a task to be done, and allows users to mark them as such.